xapian-core/tests/harness/backendmanager_remote.cc

```diff
@@ -29,16 +29,22 @@
 
 Xapian::WritableDatabase
 BackendManagerRemote::get_writable_database_from_sub_manager
-				    (const std::string & name)
+				    (const std::string& name)
 {
     return sub_manager->get_writable_database(name, std::string());
 }
 
 std::string
-BackendManagerRemote::get_generated_database_path(const std::string & name) {
+BackendManagerRemote::get_generated_database_path(const std::string& name) {
     return sub_manager->get_writable_database_path(name);
 }
 
+Xapian::WritableDatabase
+BackendManagerRemote::get_generated_database(const std::string& name)
+{
+    return get_writable_database_from_sub_manager(name);
```

Comment:
Similar to `get_alt_writable_database()`, `get_writable_database_from_sub_manager()`'s code can now be moved here and it can be eliminated.

Reply:
eliminated `get_alt_writable_database()`